packages/drivers/driver-web-cache/src/FluidCache.ts

```diff
@@ -47,6 +49,17 @@ export interface FluidCacheConfig {
 	 * If an entry exists in the cache, but is older than this value, the cached value will not be returned.
 	 */
 	maxCacheItemAge: number;
+
+	/**
+	 * To improve perf, if this property is set as false, then db will not be closed immediately after usage.
+	 * Default is true.
+	 */
+	immediateClose?: boolean;
```

Comment:
Can we have same support by saying that closeDbAfter <=0 means immediate closure?
Maybe we want to keep zero as non-special (it's delayed closed as 0ms timer has lower priority than immediate work or micro-task). Not sure - feels like we do not care.

Reply:
The reason to have one property - it's a bit ambiguous when using true / 2000 setting.